homeassistant/components/device_tracker/config_entry.py

```diff
@@ -208,6 +212,22 @@ def state_attributes(self) -> dict[str, StateType]:
         return attr
 
 
+class DeviceTrackerEntityComponent(EntityComponent[BaseTrackerEntity]):
+    """Mailbox entity component."""
+
+    def __init__(
+        self,
+        logger: logging.Logger,
+        domain: str,
+        hass: HomeAssistant,
+        scan_interval: timedelta = DEFAULT_SCAN_INTERVAL,
+    ) -> None:
+        """Initialize an entity component."""
+        super().__init__(logger, domain, hass, scan_interval)
+
+        self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, self._async_shutdown)
```

Comment:
This could be solved via #90938, but maybe it is safer to change only `device_tracker`?